src/react/components/window/index.js

```diff
@@ -125,6 +125,7 @@ const createBringToTop = (element) => {
  */
 export const showPopup = (title, content, onClose, options = {}) => {
     validate(options, TYPE.POPUP);
+
```

Comment:
Handle theme here instead by passing `options.theme` to ThemeProvider. If it's not defined it will default to the current theme and if it is defined the given one will be used.